cli/command/image/load_test.go

```diff
@@ -0,0 +1,106 @@
+package image
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/cli/internal/test"
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/docker/pkg/testutil/golden"
+	"github.com/pkg/errors"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestNewLoadCommandErrors(t *testing.T) {
+	testCases := []struct {
+		name          string
+		args          []string
+		isTerminalIn  bool
+		imageLoadFunc func(input io.Reader, quiet bool) (types.ImageLoadResponse, error)
+	}{
+		{
+			name: "wrong-args",
+			args: []string{"arg"},
+		},
+		{
+			name: "input-to-terminal",
+			isTerminalIn: true,
+		},
+		{
+			name: "pull-error",
+			imageLoadFunc: func(input io.Reader, quiet bool) (types.ImageLoadResponse, error) {
+				return types.ImageLoadResponse{}, errors.Errorf("something went wrong")
+			},
+		},
+	}
+	for _, tc := range testCases {
+		cli := test.NewFakeCli(&fakeClient{imageLoadFunc: tc.imageLoadFunc}, new(bytes.Buffer))
+		in := test.NewFakeInStream(ioutil.NopCloser(strings.NewReader("")))
```

Comment:
As said before, this shouldn't be required, just a `cli.SetIn(ioutil.NopCloser(strings.NewReader(input)))` should be enough 👼 and probably having a `fakeCli` method to set a "TerminalIn" or a non `TerminalIn`